src/server/status.rs

```diff
@@ -312,7 +312,7 @@ pub fn print_status_all(extended: bool, debug: bool, json: bool)
         let mut table = Table::new();
         table.set_format(*table::FORMAT);
         table.set_titles(Row::new(
-            ["Name", "Port", "Version", "Status"]
+            ["Name", "Port", "Version", "Method", "Status"]
```

Comment:
I'm afraid "Method: native" might confuse users who are not using docker installs. That's why I suggested to use "Installation". Also, could you please left-align table headers?